ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/metering_report_service.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1.admin;
+
+option java_outer_classname = "MeteringReportOuterClass";
+option java_package = "com.daml.ledger.api.v1.admin";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
+
+import "google/protobuf/timestamp.proto";
+
+// Experimental API to extract metering reports.
+//
+// Authorization rules for the services RPCs are specified on the `RpcNameRequest`
+// messages as boolean expressions over these facts:
+// - ``HasRight(r)``: true iff the authenticated user has right `r`
+//
+service MeteringReportService {
+
+    // Run a report.
+    rpc Report (ReportRequest) returns (ReportResponse);
```

Comment:
```suggestion
    rpc GetMeteringReport (GetMeteringReportRequest) returns (GetMeteringReportResponse);
```

Given that this service and its messages are not namespaced separately from the other `admin` services, I'd suggest structuring the names such that the core concept is the one of a `MeteringReport`, which meters activity on a participant node for billing purposes.

The alternative would be to move the service to `com.daml.ledger.api.v1.admin.metering` and drop the `Metering` prefix in the messages.

Reply:
I'm ambivalent on which one is better. I just want to make sure that we do not use the overly generic name `com.daml.ledger.api.v1.admin.ReportRequest`.

Reply:
I've changed 'Report' to 'MeteringReport' across the file to preserve the more general 'Report' name.  I've not gone for the 'Get' prefix as the reports do not really 'resource' semantics (get/put/delete) but more like pruning which uses just 'Prune'.